runtime/src/lib.rs

```diff
@@ -139,10 +141,9 @@ impl sdk::Runtime for Runtime {
 
     fn trusted_signers() -> Option<TrustedSigners> {
         #[allow(clippy::partialeq_to_none)]
-        if option_env!("OASIS_UNSAFE_SKIP_KM_POLICY") == Some("1")
-            || cfg!(feature = "debug-mock-sgx")
-        {
-            return Some(TrustedSigners::default());
+        #[cfg(feature = "debug-mock-sgx")]
+        if option_env!("OASIS_UNSAFE_SKIP_KM_POLICY") == Some("1") {
```

Comment:
This also won't work, get rid of the env check, only the debug-mock-sgx feature is relevant in our localnet builds AFAIK.